src/vuestic-theme/vuestic-sass/global/ContentClass.demo.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <VbDemo>
+    <VbContainer title="with content class (h1-h6)">
+      <div class="content">
+        <h1>h1 title</h1>
+        <h2>h2 title</h2>
+        <h3>h3 title</h3>
+        <h4>h4 title</h4>
+        <h5>h5 title</h5>
+        <h6>h6 title</h6>
+      </div>
+    </VbContainer>
+    <VbContainer title="without content class (h1-6)">
+      <div>
+        <h1>h1 title</h1>
+        <h2>h2 title</h2>
+        <h3>h3 title</h3>
+        <h4>h4 title</h4>
+        <h5>h5 title</h5>
+        <h6>h6 title</h6>
+      </div>
+    </VbContainer>
+    <VbContainer title="with content class (pre)">
+      <div class="content">
+        <pre>pre text must be here</pre>
+        <p>class: code</p>
+        <pre class="code">code must be here</pre>
+      </div>
+    </VbContainer>
+    <VbContainer title="without content class (pre)">
+      <div>
+        <pre>pre text must be here</pre>
+      </div>
+    </VbContainer>
+    <VbContainer title="with content class (lists)">
+      <div class="content">
+        <ol>
+          <li>item 1</li>
+          <li>item 2
+            <ol>
+              <li>item 2.1</li>
+              <li>item 2.2</li>
+            </ol>
+          </li>
+          <li>item 3</li>
+          <li>item 4</li>
+        </ol>
+        <ul>
+          <li>item 1</li>
+          <li>item 2
+            <ul>
+              <li>item 2.1</li>
+              <li>item 2.2</li>
+            </ul>
+          </li>
+          <li>item 3</li>
+          <li>item 4</li>
+        </ul>
+      </div>
+    </VbContainer>
+    <VbContainer title="without content class (lists)">
+      <ol>
```

Comment:
So the way it looks is off from our expectations. That should be explicitly stated.
Same story for headers.

We want demos to contain only working stuff.